packages/db-objects-redis/package.json

```
@@ -26,14 +26,19 @@
     "type": "git",
     "url": "https://github.com/ioBroker/ioBroker.js-controller/packages/db-objects-redis"
   },
-  "scripts": {},
-  "main": "index.js",
+  "scripts": {
+    "build": "tsc -b tsconfig.build.json && npm run copy-files",
+    "copy-files": "cp -r ./src/lib/objects/lua* ./build/lib/objects/"
```

```suggestion
    "build": "tsc -b tsconfig.build.json",
    "postbuild": "cp -r ./src/lib/objects/lua* ./build/lib/objects/"
```
seems a bit simpler :)

---

Ah Nice will change it in adapter too where we copy certs